src/api/rosetta-constants.ts

```diff
@@ -51,9 +51,15 @@ export const RosettaOperationTypes = [
   'mint',
   'burn',
   'miner_reward',
-  'stx_lock',
+  'stx_lock', // stx event
+  'stx_unlock', // forged event
+  'stack-stx', // PoX contract function
+  'delegate-stx', // PoX contract function
+  'revoke-delegate-stx', // PoX contract function
 ];
 
+export const PoxContractId = 'SP000000000000000000002Q6VF78.pox';
```

Comment:
This should have a testnet and mainnet constant, similar to the bns usage here:
https://github.com/blockstack/stacks-blockchain-api/blob/0565d4f7f3130ed418b4652be3ae3c2a548bffe2/src/bns-constants.ts#L17-L20